tests/phpunit/elementor/includes/template-library/test-manager-cloud.php

```diff
@@ -124,7 +120,8 @@ public function test_save_template() {
 				'templateType' => 'container',
 				'parentId' => null,
 				'content' => $mock_content,
-			]);
+			])
+			->willReturn($post_resource_response);;
```

Comment:
```suggestion
			->willReturn( $post_resource_response );
```
Not sure hwo strict we should be regarding the spacings/formatting since PHP Lint does not fail.